src/libLookup/Lookup.cpp

```diff
@@ -325,9 +326,9 @@ bool Lookup::GenTxnToSend(size_t num_txn, vector<Transaction>& txn) {
       continue;
     }
 
-    copy(txns.begin(), txns.end(), back_inserter(txn));
+    copy(txns.begin(), txns.end(), back_inserter(DSTxn));
   }
-  return !txn.empty();
+  return !(shardTxn.empty() || DSTxn.empty());
```

Comment:
should it be && ?
both shardTxn and DSTxn should be empty, right?

Reply:
Should be both not empty, so I think || is correct

Reply:
Yes, that's fine then